docs/docs/icechunk-python/configuration.md

```diff
@@ -4,16 +4,138 @@ When creating and opening Icechunk repositories, there are many configuration op
 
 !!! note
-    This section is under construction and coming soon.
 
-## Creating and Opening Repos
+    Currently only `s3` compatible storage and `local_filesystem` storage are supported for virtual chunk containers. Other storage backends such as `gcs`, `azure`, and `https` are on the roadmap.
+
+#### Example
+
+For example, if we wanted to configure an icechunk repo to be able to contain virtual chunks from an `s3` bucket called `my-s3-bucket` in `us-east-1`, we would do the following:
+
+```python
+config.virtual_chunk_containers = [
+    icechunk.VirtualChunkContainer(
+        name="my-s3-bucket",
+        url_prefix="s3://my-s3-bucket/",
+        storage=icechunk.StorageSettings(
+            storage=icechunk.s3_storage(bucket="my-s3-bucket", region="us-east-1"),
+        ),
+    ),
+]
+```
+
+If we also wanted to configure the repo to be able to contain virtual chunks from another `s3` bucket called `my-other-s3-bucket` in `us-west-2`, we would do the following:
+
+```python
+config.set_virtual_chunk_container(
+    icechunk.VirtualChunkContainer(
+        name="my-other-s3-bucket",
+        url_prefix="s3://my-other-s3-bucket/",
+        storage=icechunk.StorageSettings(
+            storage=icechunk.s3_storage(bucket="my-other-s3-bucket", region="us-west-2"),
+        ),
+    ),
+)
+```
+
+Now at read time, if icechunk encounters a virtual chunk url that starts with `s3://my-other-s3-bucket/`, it will use the storage configuration for the `my-other-s3-bucket` container.
+
+!!! note
+
+    While virtual chunk containers specify the storage configuration for any virtual chunks, they do not contain any authentication information. The credentials must also be specified when opening the repository using the [`virtual_chunk_credentials`](./reference.md#icechunk.Repository.open) parameter. See the [Virtual Chunk Credentials](#virtual-chunk-credentials) section for more information.
+
+### [`manifest`](./reference.md#icechunk.RepositoryConfig.manifest)
+
+The manifest configuration for the repository. [`ManifestConfig`](./reference.md#icechunk.ManifestConfig) allows you to configure behavior for how manifests are loaded. in particular, the `preload` parameter allows you to configure the preload behavior of the manifest using a [`ManifestPreloadConfig`](./reference.md#icechunk.ManifestPreloadConfig). This allows you to control the number of references that are loaded into memory when a session is created, along with which manifests are available to be preloaded.
+
+#### Example
+
+For example, if we have a repo that contains an [`Xarray`](./xarray.md) dataset, we may want to configure the manifest preload to only preload manifests that contain arrays that are coordinates, in our case `time`, `latitude`, and `longitude`.
```

Comment:
There technically isn't any such thing as "a repo that contains an Xarray dataset". It would be more accurate to say "if we have a repo which contains data that we plan to open as an xarray dataset..."